welcome/oce_about.adoc

```diff
@@ -0,0 +1,211 @@
+[[about_oce]]
+= About OCE
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
```

Comment:
We need a little context to introduce the table.

Reply:
I am using the intro section of the source document for this. It is markety, so feel free to change.

"Red Hat OpenShift Container Engine (OCE) is a new product offering from Red Hat and can be used if you wish to use OpenShift as a production platform for launching containers without leveraging the more advanced features in OpenShift Container Platform (OCP). OCE is the same product as OCP, in terms of download, installation, and general usage. 

OCE entitles you to use a subset of the features found in OCP and the table here highlights the differences between these two platforms."